charts/orchestrator/templates/sonata-network-policy.yaml

```diff
@@ -0,0 +1,34 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: RHDH-to-sonata-runtimes
+spec:
+  podSelector:
+    matchExpressions:
+      - { key: sonataflow.org/workflow-app, operator: Exists }
+  ingress:
+    - from:
+      - namespaceSelector:
+          matchLabels:
+            kubernetes.io/metadata.name: rhdh-operator 
+        podSelector:
+          matchLabels:
+            rhdh.redhat.com/app: backstage-backstage
+---
+
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: RHDH-to-sonata-services
+spec:
+  podSelector:
+    matchExpressions:
+      - { key: app, operator, In, values: [ sonataflow-platform ] }
+  ingress:
+    - from:
+      - namespaceSelector:
+          matchLabels:
+            kubernetes.io/metadata.name: rhdh-operator 
+        podSelector:
+          matchLabels:
+            rhdh.redhat.com/app: backstage-backstage
```

Comment:
this will not be applied to an existing backstage instances with a different name than `backstage` (this CR name is configurable).
therefore we should add a section about applying those resources to https://github.com/parodos-dev/parodos-dev.github.io/blob/main/content/blog/installation-on-existing-rhdh.md

Reply:
Or why not create a new label? If we add some doc for already existing instances, we could have a new label.
And for new instances, the label would be applied automatically

Reply:
we can't set that lable on the RHDH because we are not resposible for it 

unfortuntely rhdh do not set a label which is independant from the CR name that can identify backstage only
i can check the existnace of the key but that may catch more componednt than needed.
i'll double check again

Reply:
we create the Backstage CR [here](https://github.com/parodos-dev/orchestrator-helm-chart/blob/main/charts/orchestrator/templates/rhdh-operator.yaml#L4) but not sure any label will be propagated to the pods. we can ask for an RFE for that (similar to what we did with sonataflow)